frontend/src/libs/components/card/styles.module.scss

```
@@ -0,0 +1,43 @@
+.chat-element {
```

rename classes without chat
```suggestion
.item {
```

---

done